database/seeders/PostLayoutTableSeeder.php

```diff
@@ -13,7 +13,7 @@ class PostLayoutTableSeeder extends Seeder
      */
     public function run(): void
     {
-        $layouts = ['abilities', 'assets', 'attributes', 'connection_map', 'inventory', 'character_orgs', 'quest_elements', 'location_characters'];
+        $layouts = ['abilities', 'assets', 'attributes', 'connection_map', 'inventory', 'character_orgs', 'quest_elements', 'location_characters', 'entity_events'];
```

Comment:
Can you rename the key reminders, to make our life easier when later this year we rename the module to Reminders